cardano-api/src/Cardano/Api/Address.hs

```diff
@@ -32,6 +33,7 @@ module Cardano.Api.Address (
     shelleyAddressInEra,
     anyAddressInShelleyBasedEra,
     anyAddressInEra,
+    toAddressAny,
```

Comment:
Put this in the previous patch, where it gets defined.

Reply:
Done.